science/gnss-sdr/Portfile

```diff
@@ -25,14 +28,16 @@ if {${subport} eq "gnss-sdr"} {
     checksums           rmd160 733c3211b163825be36db80d89f9fb0db0051264 \
                         sha256 d44b32fd2bbdc703097e2368281d77ad4e2c42ec7c76c6e7ef100b014a716e3e \
                         size   3575803
-    revision            1
+    revision            2
 
     conflicts           gnss-sdr-devel gnss-sdr-next
 
     depends_lib-append  port:gnuradio port:volk-gnss-sdr
 
     require_active_variants port:gnuradio uhd
 
+    patchfiles          patch-boost-asio.diff patch-fix-gnuradio-version.diff
```

Comment:
Please split these into 2 patches (use "-append" if/as necessary), then add a brief comment to each about as to what it's for / doing

Reply:
Done.